.github/CODEOWNERS

```diff
@@ -640,6 +640,7 @@ peps/pep-0756.rst  @vstinner
 peps/pep-0757.rst  @vstinner
 peps/pep-0758.rst  @pablogsal @brettcannon
 peps/pep-0789.rst  @njsmith
+peps/pep-0760.rst  @pablogsal @brettcannon
```

Comment:
Put this before 789.

Reply:
Rebase and you'll get an entry for PEP 759.